docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -47,6 +48,7 @@ Configuration template:
 
 - `min_part_size` – The minimum size of a data part.
 - `min_part_size_ratio` – The ratio of the data part size to the table size.
+- `enable_deflate_qpl_codec` – The control of deflate_qpl codec support. If intend to use deflate_qpl as compression method, please set it to be true, othewise false or just omit.
```

Comment:
This will be sufficient as an explanation: `Controls if compression codec "deflate_qpl" is enabled (it is disabled by default).`

Reply:
If you don't mind, can you please add a corresponding (commented out) "enable_deflate_qpl_codec" tag to file programs/server/config.xml and programs/server/config.yaml.example? As far as I know, this file is shipped with installations as the default configuration.

Reply:
Done